model/src/main/kotlin/utils/VersionUtils.kt

```diff
@@ -63,4 +63,4 @@ internal fun Identifier.isApplicableIvyVersion(pkgId: Identifier) =
         it.showStackTrace()
     }.getOrDefault(false)
 
-private fun String.isVersionRange() = versionRangeIndicators.any { contains(it, ignoreCase = true) }
+private fun Identifier.isVersionRange() = versionRangeIndicators.any { version.contains(it, ignoreCase = true) }
```

Comment:
In the previous commit I proposed to operate on strings for the other function, so that comment of mine kind of conflicts with this commit. However, I'm ok if you should prefer continue to operate on Identifier, but please consider my other comment to decide, up to you.